README.md

```diff
@@ -31,11 +31,7 @@ That said, keep in mind the following:
 
 ## Setup
 
-Get the tool via
-
-```sh
-$ go get -u github.com/golang/dep/cmd/dep
-```
+Grab the latest binary from the [releases](https://github.com/golang/dep/releases) page.
```

Comment:
i don't think we can remove it, but we should de-emphasize it - put it after other recommended methods.